cicecore/cicedynB/dynamics/ice_dyn_shared.F90

```diff
@@ -1378,13 +1378,13 @@ subroutine seabed_stress_factor_prob (nx_block, ny_block,          &
            TbN         ! seabed stress factor at N location (N/m^2)
 
       integer (kind=int_kind), intent(in), optional :: &
-           icellE, icellN ! no. of cells where ice[en]mask = 1
+           icelle, icelln ! no. of cells where ice[en]mask = 1
 
       integer (kind=int_kind), dimension (nx_block*ny_block), intent(in), optional :: &
-           indxEi  , & ! compressed index in i-direction
-           indxEj  , & ! compressed index in j-direction
-           indxNi  , & ! compressed index in i-direction
-           indxNj      ! compressed index in j-direction
+           indxei  , & ! compressed index in i-direction
+           indxej  , & ! compressed index in j-direction
+           indxni  , & ! compressed index in i-direction
```

Comment:
I think the capitalization helps with interpreting the variables when reading the code, because it indicates grid locations.  E.g. we use lower-case n for thickness categories, and I've always used lower-case t for time and lower-case u (and v) for velocity.  But I won't quibble if others prefer lower case for these.

Reply:
I go back and forth.  I changed a bunch of variables names a few months back in ice_dyn_shared, but they are not consistent with the rest of the code now.  I guess I thought we might change them in other places, but that hasn't happened.  I think we should move to capital T, U, N, E, but I think consistency in the code is generally more important, so I changed some of these back.  There is a lot we could still do to clean this up if we wanted.

Reply:
If we ultimately want to move to capitals for these kinds of things, then I'd advocate for keeping the capitals here and, whenever there's time/energy (not necessarily this PR), fix the others to be consistent.

Reply:
OK, I'll undo these changes before we merge, no problem.